ipv8/test/test_taskmanager.py

```diff
@@ -34,22 +33,22 @@ def tearDown(self):
         self.tm.shutdown_task_manager()
         super(TestTaskManager, self).tearDown()
 
-    @twisted_wrapper
+    @inlineCallbacks
     @untwisted_wrapper
```

Comment:
Do we still need the `inlineCallbacks` and `untwisted_wrapper` with trial?

Reply:
I don't know.
Like, I really tried to minimize the invasiveness of the changes. The plan was to remove the cruft after this PR is merged and we already have working Trial-based tests.

Reply:
Fair enough.